src/porepy/numerics/elliptic.py

```diff
@@ -43,7 +44,7 @@ class Elliptic():
             Returns Flux discretization object (E.g., Tpfa)
     grid(): Returns: the Grid or GridBucket
     data(): Returns: Data dictionary
-    split(name): Assignes the pressure self.p to the data dictionary at each
+    split(name): Assignes the solution self.solution to the data dictionary at each
```

Comment:
should it be self.x and not self.solution ?

Reply:
yes, you're right.